src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -86,8 +86,9 @@ MavlinkMissionManager::MavlinkMissionManager(Mavlink *mavlink) : MavlinkStream(m
 	_offboard_mission_sub(-1),
 	_mission_result_sub(-1),
 	_offboard_mission_pub(nullptr),
-	_slow_rate_limiter(_interval / 5.0f),
-	_verbose(false)
+	_slow_rate_limiter(100 * 1000), // Rate limit sending of the current WP sequence to 10 Hz
```

Comment:
This rate limiter is only applied for sending the current waypoint. i.e. during a mission, when the vehicle reaches a waypoing, this will change, and QGC can update the currently marked waypoint.